src/components/sale/OrderExportModal.tsx

```diff
@@ -177,6 +178,7 @@ const OrderExportModal: React.FC<AdminModalProps> = ({ renderTrigger, ...adminMo
             orderLog.order_discounts?.split('\\n').join('\n') || '',
             orderLog.order_product_num || 0,
             Math.max(orderLog.order_product_total_price || 0, 0),
+            enabledModules.merchandise ? orderLog.shipping?.fee || 0 : undefined,
```

Comment:
shipping fee shuld be decide by product is merchandise
if product is not merchandise, display empty string
btw, undefined will display 0

Reply:
resolved